<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorFrame</class>
 <widget class="QMainWindow" name="EditorFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GalaxyEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewScene"/>
    <addaction name="actionOpenScene"/>
    <addaction name="separator"/>
    <addaction name="actionSaveScene"/>
    <addaction name="actionSaveAsScene"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionLoad_Layout"/>
    <addaction name="actionSaveLayout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Window"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenScene">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSaveLayout">
   <property name="text">
    <string>&amp;Save Layout</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionLoad_Layout">
   <property name="text">
    <string>&amp;Load Layout</string>
   </property>
  </action>
  <action name="actionSaveScene">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionNewScene">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionSaveAsScene">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="editorqt.qrc"/>
 </resources>
 <connections/>
</ui>
